### DISCUSSION TITLE
Error message in the console, still generating images.

### DISCUSSION THREAD

**Discussion by BlastedRemnants** [adieyal/sd-dynamic-prompts#87] [General]
I've noticed a new error message in the console window that seems connected to Dynamic Prompts somehow, but everything seems to be working fine so I didn't want to start an issue. In any case here's the error message I noticed, maybe a new feature that isn't quite working yet? Looks like a negative prompt maker, could be an interesting feature for sure.

Error running process: C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py
Traceback (most recent call last):
  File "C:\SDWebUI\modules\scripts.py", line 327, in process
    script.process(p, *script_args)
  File "C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py", line 350, in process
    generator = self._create_generator(
  File "C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py", line 130, in _create_generator
    generator = old_generation(
  File "C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py", line 78, in old_generation
    prompt_generator = RandomPromptGenerator(
TypeError: RandomPromptGenerator.__init__() takes from 3 to 4 positional arguments but 5 were given

Error running process_batch: C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py
Traceback (most recent call last):
  File "C:\SDWebUI\modules\scripts.py", line 336, in process_batch
    script.process_batch(p, *script_args, **kwargs)
  File "C:\SDWebUI\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py", line 302, in process_batch
    generator = self._negative_prompt_generator
AttributeError: 'Script' object has no attribute '_negative_prompt_generator'

**Comment by adieyal** [comment]
Thanks for reporting this. I can't re-create the bug. Are you running the latest version of the extension? Should be 0.29.8

**Reply by BlastedRemnants** [reply]
> Thanks for reporting this. I can't re-create the bug. Are you running the latest version of the extension? Should be 0.29.8

I can no longer recreate it either, I must've been behind somehow, or I didn't update properly. Usually after I hit Apply in the WebUI's extensions page I give it a couple seconds and just reload that browser tab. Seems like that doesn't do a proper job tho, sorry!

**Comment by crackfoo** [comment]
Did you close and restart webgui? I got this but only after doing the
update and didn’t restart the whole gui.

**Reply by BlastedRemnants** [reply]
I think that's partly the problem, I do relaunch the WebUI when I update but I usually just wait a few seconds then go to my home page and reload from there, maybe I'm waiting too long or not long enough, I'll adjust and see what works best. For now tho it's gone away and I don't get the error anymore. Thanks for the response!

**Reply by adieyal** [reply]
Please re-open if it happens again

**Reply by BlastedRemnants** [reply]
Thanks, I will. And thanks again for the script, it's soooo handy and I love being able to save all my fave prompts as a wildcard. Ooh actually, quick suggestion but maybe you could add functionality that to your script's description? I don't know if many people realize that they can make their own lists like you have in your Presets folder. For me that was a game-changer, once I figured out that all I had to do was separate my prompts with a return and a hashtag and save them in a list to be called later with a single word, now I can launch SD and start cranking out awesome pics with one __word__, absolutely epic stuff. Just a thought tho, seems like an amazing feature that probably gets missed by most folks.

**Comment by adieyal** [comment]
Glad you're enjoying it and thanks for the suggestion. I've been toying with the idea of creating an optional tab where you can edit and create your own wildcard files within the webui.

**Reply by adieyal** [reply]
There is a new Wildcards Manager folder for exploring and editing wildcards in the new v1.0 version

**Comment by adieyal** [comment]
Could you please mark this question as answered - otherwise it's hard for me to track unanswered questions.
